aspnetcore/fundamentals/app-state/sample/src/WebAppSessionDotNetCore2.0App/WebAppSession.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <AssemblyName>WebAppSession</AssemblyName>
+    <OutputType>Exe</OutputType>
```

Comment:
An output type of exe is implied by the Web SDK. This node can be removed.

Reply:
fixed